lib/seahorn/BvOpSem2FatMemMgr.cc

```diff
@@ -155,11 +199,13 @@ class FatMemManager : public OpSemMemManager {
 
   /// \brief Returns sort of memory-holding register for an instruction
   Expr mkMemRegisterSort(const Instruction &inst) const {
-    return sort::structTy(m_mem.mkMemRegisterSort(inst));
+    return sort::structTy(m_main.mkMemRegisterSort(inst),
+                          m_slot0.mkMemRegisterSort(inst),
+                          m_slot1.mkMemRegisterSort(inst));
   }
 
   /// \brief Returns a fresh aligned pointer value
-  FatPtrTy freshPtr() override { return mkFatPtr(m_mem.freshPtr()); }
+  FatPtrTy freshPtr() override { return mkFatPtr(m_main.freshPtr()); }
```

Comment:
this is correct and a good example for all other functions that must return a **fat** pointer, but do not have particular value for the **fat**

Reply:
ack